src/api/SessionManager.ts

```diff
@@ -29,6 +29,10 @@ export default class SessionManager {
     else return null
   }
 
+  async endCurrentSession(stopCasting: boolean): Promise<void> {
```

Comment:
I think the parameter could be optional, defaulting to false.

```suggestion
  async endCurrentSession(stopCasting: boolean = false): Promise<void> {
```